scripts/grok1_multiblock_v4_supervisor.py

```
@@ -0,0 +1,2415 @@
+import traceback
+from collections.abc import Sequence
+from contextlib import contextmanager
+
+# Fixed interpreter/script argv; the call below always disables the shell.
+import subprocess  # nosec B404
+import sys
+import tempfile
+from dataclasses import dataclass
+from datetime import datetime, timezone
+from pathlib import Path
+from typing import Any, Iterator
```

Ruff (0.16.2)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)